Credits.md

```diff
@@ -0,0 +1,25 @@
+##Credits
+
+Freddy Rice - Original Beartime
+
+### BearStatus Development Team
+
+BearStatus was created by the Blake School Fall 2013 Computer Science class, taught by Sean Hickey. Continuing development and promotion is being done by Ben Weinshel and Jason Shu '15.
+
+Frontend Development: Allison Eastham '14, Jason Shu '15, Teddy Ankeny '14, Pierce Carlson '15
+
+Backend Development: Kevin Hayes '14, Laura DeMane '15, Anthony Hu '15, Marcus Berg '14
+
+Integration: Noah Abramovitz '14, Yona Cooperman '15, Ben Weinshel '15, Brynne Swearingen '15
+
+Project Management: Walker Bond '15, Quinn Ellis '15, Carolyn Nye '15, Meghna Kaul '14
+
+### Beartime Web
```

Comment:
I would suggest leading with information about the current version. Then lower down the page we can credit those who worked on past versions.